internal/service/privatelinkendpointserverless/resource_privatelink_endpoint_serverless.go

```diff
@@ -34,6 +35,7 @@ func Resource() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			StateContext: resourceImport,
 		},
+		DeprecationMessage: fmt.Sprintf(constant.DeprecationResourceByDateWithExternalLink, "March 2025", "placeholder-serverless-deprecation-url"),
```

Comment:
are we waiting to merge the PR until we know the URL?

Reply:
I will wait to merge at when at least we have a draft for CLOUDP-280488 so that we can put the link for the guides created there (of course the guides won't exist until release, but we can know the URL they will have knowing the name of the file)

Reply:
draft is available https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2746